# 인사

안녕하세요 ~~ 25기 여러분 반갑습니다 ~~  

드디어 정규세션이 시작되었네요.  

뭘 과제로 내볼까 고민하다가, 요즘 데이터 분석 채용 공고 자격 요건에 필수로 들어있는.... A/B TEST가 좋겠다!!고 생각했습니다.  

데이터 분석쪽이 아니더라도, 다양한 도메인에서 쓸 수 있는 방법이라고 생각해요.


A/B TEST의 개념부터 기초적인 과정을 이해해볼 수 있게 내봤어요. 근데 설명해야할 게 많다보니 텍스트가 너무 많아졌네요.

읽기 힘들어도 ㅎㅎ... 열심히 해주시고, 질문 사항은 저에게 연락 남겨주세요.  

참고로, 정규세션 발표 외의 내용을 구글링하고 공부해야만 풀 수 있는 문제들이 포함되어 있습니다.

제 이름... 아시죠...? 24기 박현서입니다....



<br>

# Q1. A/B TEST란 무엇일까요?   
또한, A/B TEST시 유의할 사항들을 초두효과를 포함해 기술해주세요.

정규세션에서 발표한 내용 외에 추가적으로 공부한 개념을 작성하셔도 됩니다.
정의와 함께 예시 하나만 들어주세요 ~~

## A1. 답안 작성  

### A/B TEST 정의
A/B TEST란 두 가지 이상의 버전(A: 대조군, B: 실험군)을 무작위로 나눈 사용자 집단에 동시에 노출시킨 뒤, 사전에 정의한 핵심 지표(KPI)를 통계적으로 비교하여 어느 버전이 더 우수한 성과를 내는지 데이터 기반으로 판단하는 통제된 실험 방법이다.

### 예시
넷플릭스가 콘텐츠 썸네일 이미지의 효과를 비교하는 경우를 들 수 있다. 동일한 영화에 대해 A그룹에는 기존 포스터형 썸네일을, B그룹에는 주인공의 감정 표현이 잘 드러나는 스틸컷 썸네일을 노출시킨 뒤, 콘텐츠 클릭률(CTR)을 비교하여 어떤 이미지가 더 높은 클릭을 유도하는지 검증한다.

### A/B TEST 시 유의사항

1. **초두효과(Primacy Effect)**: 사용자가 기존 UI에 익숙해져 있어 새로운 변경사항에 대해 초기 거부감이나 반대로 신선함에 의한 일시적 행동 변화가 발생할 수 있다. 따라서 실험 초기 데이터만으로 성급하게 판단하지 말고, 충분한 기간 동안 데이터를 수집하여 초두효과가 안정화된 이후의 데이터를 기반으로 분석해야 한다.

2. **무작위 배정(Randomization)**: 대조군과 실험군에 사용자를 무작위로 배정하여 선택 편향(Selection Bias)을 제거해야 한다. 특정 시간대, 지역, 디바이스 등에 치우친 배정은 결과를 왜곡할 수 있다.

3. **충분한 표본 크기**: 통계적으로 유의미한 결과를 도출하기 위해 사전에 필요한 최소 표본 크기를 계산(Power Analysis)하고, 이를 충족해야 한다.

4. **단일 변수 통제**: 한 번에 하나의 변수만 변경하여 테스트해야 결과의 인과관계를 명확히 파악할 수 있다. 여러 변수를 동시에 바꾸면 어떤 변경이 효과를 만들었는지 구분이 불가능하다.

5. **노벨티 효과(Novelty Effect)**: 새로운 UI에 대한 호기심으로 초기에 지표가 일시적으로 상승할 수 있으므로, 이 효과가 사라진 후의 안정적인 데이터를 확보하는 것이 중요하다.

6. **동시 실험(Simultaneous Exposure)**: A/B 두 그룹을 동일한 기간에 동시 노출해야 하며, 시간차를 두면 외부 요인(시즌, 이벤트 등)에 의한 교란이 발생할 수 있다.



---



 A/B TEST를 하기 위해서는 1-pager를 작성해야합니다.  

1-pager란 문제 정의, 관찰하고자하는 실험대상, 가설설정, 평가지표 등을 작성하는  
간단히 말해 계획서를 의미하는데요,

다른 부서와의 소통을 위한 역할 또,과거에 어떤 테스트를
어떻게 진행했는지에 대한 기록의 역할을 하기도 합니다.


 A/B TEST의 1-pager는 복잡한 실험 과정을 한눈에 파악하고, 의사결정권자가 "그래서 이 실험을 왜 했고, 결과가 무엇이며, 무엇을 할 것인가?"를 바로 이해할 수 있도록 작성하는 것이 핵심입니다.

예시를 통해 어떤 항목이 있고, 각 항목에 어떤 내용을 채우는 건지 보여드릴게요.



---



**예시**   
(푸는 거 아니고, 예시입니다! 읽으면서 이런 부분은 이런 내용을 채우는 거군... 이해하시면 돼요!)  




 📊 [ThinkGood] 메인 홈 검색 모듈 상위 노출 실험 1-Pager

---

**1. 실험 전략 및 배경 (Strategy & Observation)**

| 항목 | 내용 |
| :--- | :--- |
| **Target User** | 특정 공모전을 찾으려는 목적이 뚜렷하지만, 현재 하단에 위치한 검색바를 즉각적으로 발견하지 못한 유저 |
| **Observation** | **[데이터 분석 결과]** <br> • 검색 이용 유저의 상세 진입 전환율(CVR)은 40%, 미이용 유저의 전환율은 25%임 <br> • 현재 검색 이용률은 30%로, 검색 가시성 개선을 통한 전환율 상승 여력이 큼 |
| **Problem Statement** | 검색바가 배너 하단에 위치하여 시인성이 낮으며, 유저가 정보를 탐색하기 전 이탈할 확률이 높음 |
| **Hypothesis** | 하단 검색바를 페이지 **최상단으로 노출**시켜 검색 접근성을 높인다면, 전체 공모전 상세 진입률(CVR)이 개선될 것이다 |

---

 **2. 실험군 설계 (Experimental Groups)**

<table>
  <tr>
    <td align="center" width="50%">
      <b>A (Control) - 기존안</b><br>
      <img src="https://i.postimg.cc/kgcLc7Qy/Gemini-Generated-Image-enqvuhenqvuhenqv-(1).png" width="100%"><br>
      <sub>검색바 하단 위치 (기존 UI)</sub>
    </td>
    <td align="center" width="50%">
      <b>B (Variant) - 실험안</b><br>
      <img src="https://i.postimg.cc/0N51Rz5M/Gemini-Generated-Image-enqvuhenqvuhenqv.png" width="100%"><br>
      <sub>검색바 최상단 배치 (변경 UI)</sub>
    </td>
  </tr>
</table>

---

 **3. 지표 및 통계 설계 (Metrics & Statistics)**

> **핵심 측정 지표**
> * **Primary Metric (주지표):** 공모전 상세 페이지 진입률 (Detail Page CVR)
> * **Secondary Metric (보조지표):** 검색바 클릭률 (Search CTR)
> * **Guardrail Metric (방어지표):** 메인 페이지 이탈률 (Bounce Rate)

| 통계적 설계 항목 | 설정값 | 비고 |
| :--- | :--- | :--- |
| **Significance Level ($\alpha$)** | 5% | 유의수준 0.05 |
| **Statistical Power ($1-\beta$)** | 80% | 검정력 유지 |
| **Minimum Sample Size** | **총 92,000명** | 각 그룹당 46,000명 할당 |
| **Required Period** | **최소 4일** |  

---

**4. 위험 관리 및 중단 기준 (Risk & Andon)**

* **Trade-Off:**   
    검색바 상단 노출로 인해 메인 배너의 시각적 주목도가 분산될 수 있으며, 기존 UI에 익숙한 유저에게 일시적인 초두 효과(Primacy Effect)에 의한 혼란이 발생할 수 있음.
* **Andon (중단 기준):**
    1. 대조군 대비 전체 상세 진입률이 **5% 이상 유의미하게 하락**할 경우
    2. UI 변경으로 인한 페이지 오류나 유저 이탈률이 **3일 연속 급증**할 경우

---







<br>

# Q2. 1-pager를 작성해봅시다. (꼬리문제 3 + 보고서 채우는 것1)
저희가 만든 데모 사이트 링크를 첨부해드릴테니, 마음껏 보신 뒤에 위의 예시처럼 1-pager를 작성해보기 바랍니다.

여기서 실험군 설계에 들어가는 사진은 굳이 넣을 필욘 없고 말로만 설명해주셔도 됩니다. 아니면 gemini에게 요청해볼 수도 있겠네요!  

또한 문제탐색에 필요한 cvr과 같은 지표들은 직접 가정하고 하시면 됩니다. 다만, 해당 지표들이 큰 문제가 있는 지표여야... a/b test를 진행하는 이유가 되겠죠!?


정리해보면, 저희 웹사이트 보시고 '이건 여기에 배치하면 더 좋을 거 같은데?' 하는 것들을 찾아서 모든 것을 가정한 뒤에 쓰시면 됩니다!

[데모 웹사이트](https://huggingface.co/spaces/HanJaeHui/ThinkGood_Recommendation_System)


추가적으로 아래 실험군 항목에 사진을 넣으실 분들은 제가 아래 첨부드리는 사이트에 방문하셔서 해당 이미지를 코랩에 넣을 수 있는 형태로 변환한 링크를 복붙하시면 됩니다.  

(.png로 끝나는 링크 복붙하시면 됩니다.)

[포스트 이미지 사이트](https://postimages.org/)



<br>

## Q2-1. 실험군을 설계 해봅시다.
아래 보고서를 채워 주세요.

## A2-1. 아래 보고서 항목 설명을 참고하여 '[ ]' 부분을 지우고 채워주세요.  
  



📊 [ThinkGood] 공모전 추천 카드 CTA 버튼 개선 실험 1-Pager

---

1. 실험 전략 및 배경 (Strategy & Observation)

| 항목 | 내용 |
| :--- | :--- |
| **Target User** | ThinkGood 메인 페이지에서 추천 공모전 카드를 열람하지만, 상세 페이지로 진입하지 않고 이탈하는 유저 |
| **Observation** | **데이터 근거** <br> • 추천 공모전 카드 노출 대비 상세 페이지 진입률(CVR)이 12%에 그치고 있음 <br> • 현재 추천 카드에는 명확한 CTA(Call-To-Action) 버튼이 없고, 카드 전체 클릭으로만 진입 가능하여 유저가 클릭 가능 여부를 인지하지 못하는 경우가 많음 |
| **Problem Statement** | 추천 공모전 카드에 시각적으로 명확한 행동 유도 요소가 없어, 유저가 관심이 있더라도 상세 페이지로 진입하지 않고 스크롤만 하다 이탈하는 문제가 발생하고 있음 |
| **Hypothesis** | 추천 공모전 카드 하단에 **"자세히 보기" CTA 버튼을 추가**하면, 유저의 클릭 유도가 강화되어 상세 페이지 진입률(CVR)이 개선될 것이다 |

---

 2. 실험군 설계 (Experimental Groups)

<table>
  <tr>
    <td align="center" width="50%">
      <b>A (Control) - 기존안</b><br>
      <sub>추천 공모전 카드: 공모전명, 주최기관, D-day 정보만 표시. 별도의 CTA 버튼 없이 카드 전체 영역 클릭 시 상세 페이지로 이동하는 기존 UI</sub>
    </td>
    <td align="center" width="50%">
      <b>B (Variant) - 실험안</b><br>
      <sub>추천 공모전 카드 하단에 강조색(Primary Color)의 "자세히 보기 →" CTA 버튼을 추가한 변경 UI. 버튼 호버 시 색상 변경 효과 적용</sub>
    </td>
  </tr>
</table>

<br>

## Q2-2. 실험에 필요한 최소 test users 수를 구해봅시다.
유의수준 0.05, 검정력 80% 하에 수행해주세요.  


아래 공식 참고해서, 최소 test users 수를 구하는 코드 작성해주세요.

추가적으로 n은 산출하는데에 가정한 p1, p2도 같이 작성해주세요.

$$n = \frac{(Z_{\alpha/2} + Z_{\beta})^2 \cdot [p_1(1-p_1) + p_2(1-p_2)]}{(p_1 - p_2)^2}$$

## A2-2. 아래 질문에 대한 답과 코드를 채워주세요.

p1, p2는 각각 무엇을 의미하는 확률이고, 어떤 값으로 가정하셨나요?   
또한 위 공식을 보고 n을 구하기 위한 코드를 작성해주세요. 단, 라이브러리는 z값을 구하는 용도의 것만 사용해주세요.


### 답변:
- **p1**: 대조군(A, Control)의 상세 페이지 진입률(CVR). 현재 추천 카드의 CVR이 12%이므로 **p1 = 0.12**로 가정.
- **p2**: 실험군(B, Variant)의 상세 페이지 진입률(CVR). CTA 버튼 추가로 CVR이 15%로 개선될 것으로 기대하여 **p2 = 0.15**로 가정.
- 즉, 3%p의 절대적 향상(상대적으로 약 25% 개선)을 감지하기 위한 최소 표본 크기를 산출한다.

In [2]:
import math
from scipy.stats import norm

# 가정한 전환율
p1 = 0.12  # 대조군 CVR
p2 = 0.15  # 실험군 CVR (기대)

# 유의수준, 검정력
alpha = 0.05
power = 0.80
beta = 1 - power  # 0.20

# Z값 계산 (scipy.stats.norm 사용)
z_alpha_2 = norm.ppf(1 - alpha / 2)  # 양측검정 Z_{alpha/2}
z_beta = norm.ppf(1 - beta)           # Z_{beta}

# 최소 표본 크기 공식
numerator = (z_alpha_2 + z_beta) ** 2 * (p1 * (1 - p1) + p2 * (1 - p2))
denominator = (p1 - p2) ** 2

n = math.ceil(numerator / denominator)

print(f"Z_alpha/2 = {z_alpha_2:.4f}")
print(f"Z_beta    = {z_beta:.4f}")
print(f"p1 = {p1}, p2 = {p2}")
print(f"그룹당 최소 표본 크기 n = {n}명")
print(f"총 최소 표본 크기 = {n * 2}명")

Z_alpha/2 = 1.9600
Z_beta    = 0.8416
p1 = 0.12, p2 = 0.15
그룹당 최소 표본 크기 n = 2033명
총 최소 표본 크기 = 4066명


<br>

## Q2-3. 실험에 필요한 최소 실험 기간을 구해봅시다.

이 실험에 필요한 최소 test users가 92000명이 나왔고  
해당 웹페이지 하루 평균 방문자수가 50000명 정도라고 할 때 우리는 최소 4일이 필요합니다.

비교하고 싶은 대상이 총 두개이고, 하나의 대상을 실험하기 위해선 최소 이틀은 필요하기 때문이죠!

위에서 구하신 test users수를 바탕으로, 하루 평균 방문자수를 가정한 뒤 기간을 설정해주세요.  
구하신 기간은 밑에 표에 기재해주시면 됩니다.



## A2-3. 아래 세질문에 답하신 뒤 아래 보고서를 채워주세요.

가정한 하루 평균 방문자 수? **1,500명**  

산출된 최소 test users 수? **총 4,066명** (그룹당 2,033명)
  
그래서 필요한 최소 기간은? **최소 4일** (4,066 ÷ 1,500 ≈ 2.7일이지만, 두 그룹을 비교하므로 최소 2일 × 2 = 4일 필요)

## **아래 보고서도 채워주세요!**  
**'[ ]' 부분을 지우고 해당 내용에 부합하는 내용들을 채워주시면 됩니다.**

---

**3. 지표 및 통계 설계 (Metrics & Statistics)**

> **핵심 측정 지표**
> * **Primary Metric (주지표):** 추천 공모전 카드 → 상세 페이지 진입률 (Detail Page CVR)
> * **Secondary Metric (보조지표):** 추천 카드 CTA 버튼 클릭률 (CTA Click Rate)
> * **Guardrail Metric (방어지표):** 메인 페이지 평균 체류 시간 및 이탈률 (Bounce Rate)

| 통계적 설계 항목 | 설정값 | 비고 |
| :--- | :--- | :--- |
| **Significance Level ($\alpha$)** | 5% | 유의수준 0.05 |
| **Statistical Power ($1-\beta$)** | 80% | 검정력 유지 |
| **Minimum Sample Size** | **총 4,066명** | 그룹당 2,033명 할당 |
| **Required Period** | **최소 4일** | 하루 평균 방문자 1,500명 기준 |

---

 **4. 위험 관리 및 중단 기준 (Risk & Andon)**

* **Trade-Off :**    
    CTA 버튼 추가로 카드 디자인이 다소 복잡해질 수 있으며, 기존 UI에 익숙한 유저에게 초두효과(Primacy Effect)로 인한 일시적 혼란이 발생할 수 있음. 또한 버튼 영역이 추가되면서 카드 높이가 증가하여, 한 화면에 보이는 추천 카드 수가 줄어들 수 있음.

* **Andon (중단 기준) :**
    1. 대조군 대비 상세 페이지 진입률(CVR)이 **3% 이상 유의미하게 하락**할 경우
    2. CTA 버튼 도입 후 메인 페이지 이탈률(Bounce Rate)이 **3일 연속 10% 이상 증가**할 경우

---

<br>

# Q3. A/B TEST를 수행해봅시다.

1-pager를 작성했으면 직접 test도 수행해봐야겠죠!  
위에 제가 만들었던 1-pager를 바탕으로 검정을 해볼게요.  
대표적인 검정방법에는 t-test가 있고 카이제곱 겁정이 있습니다.  


---



[실험 가설]  
귀무가설 ($H_0$): 메인 홈 검색 모듈의 위치(하단 vs 최상단)와 상세 페이지 진입 여부 사이에는 상관관계가 없다. 즉, 두 그룹의 상세 페이지 진입률($CVR$)은 동일하다.$$H_0: p_A = p_B$$대립가설 ($H_1$): 메인 홈 검색 모듈을 최상단에 노출하는 것은 상세 페이지 진입 여부에 영향을 미친다. 즉, 실험군(B)의 상세 페이지 진입률이 대조군(A)보다 높을 것이다.$$H_1: p_A < p_B$$


<br>

## Q3-1. 위 가설에 적합한 검정 이름과 왜 해당 검정을 써야하는지 이유도 작성해주세요. (t test와 카이제곱검정 중)

## A3-1.  검정 이름과 이유를 설명해주세요.

답변: 위 가설에 필요한 검정은 (**카이제곱 검정 (Chi-squared Test)**) 이고,  

이유는 ```종속변수가 '상세 페이지 진입 여부(진입/미진입)'라는 범주형(이진) 변수이며, 독립변수도 '그룹(A/B)'이라는 범주형 변수이기 때문이다. 결국 두 범주형 변수 간의 독립성(연관성)을 검정하는 문제이므로 카이제곱 독립성 검정이 적합하다. t-test는 연속형 종속변수(평균)의 차이를 검정할 때 사용하는 것으로, 전환 여부(0/1)와 같은 비율 데이터를 비교하는 본 실험에는 적합하지 않기 때문``` 이다.

<br>

In [3]:
'''''''''''''''''''''''''''''''''''''''
제가 세운 A/B TEST 설계 실험 코드입니다.
'''''''''''''''''''''''''''''''''''''''

import pandas as pd
from scipy.stats import chi2_contingency

# 1. 관측 데이터 입력 (실제 실험 결과 수치로 변경하세요)
# [상세진입 성공 수, 상세진입 실패 수]
control_results = [11500, 34500]  # 대조군(A)
variant_results = [12190, 33810]  # 실험군(B)

# 2. 분할표(Contingency Table) 생성
data = [control_results, variant_results]

# 3. 카이제곱 검정 수행
chi2, p_value, dof, expected = chi2_contingency(data)

# 4. 결과 출력
print(f"카이제곱 통계량: {chi2:.4f}")
print(f"P-value: {p_value:.10f}")

alpha = 0.05
if p_value < alpha:
    print(f"결과: P-value가 {alpha}보다 작으므로 귀무가설을 기각합니다.")
    print("=> 검색바 최상단 노출이 상세 페이지 진입률에 유의미한 변화를 주었습니다.")
else:
    print(f"결과: P-value가 {alpha}보다 크므로 귀무가설을 기각할 수 없습니다.")
    print("=> 검색바 위치 변경의 효과가 통계적으로 증명되지 않았습니다.")

카이제곱 통계량: 26.9884
P-value: 0.0000002047
결과: P-value가 0.05보다 작으므로 귀무가설을 기각합니다.
=> 검색바 최상단 노출이 상세 페이지 진입률에 유의미한 변화를 주었습니다.


실제 데이터가 없어서, 저도 필요한 데이터는 가정했답니다.  

이제 25기 분들의 A/B TEST를 수행해보시죠!

<br>

## Q3-2. Q2에서 작성한 실험 수행을 위해 가설을 설정해주세요.


## A3-2. 아래 두 가설 내용을 채워주세요.
  
[실험 가설]  
귀무가설 ($H_0$): 추천 공모전 카드의 CTA 버튼 유무(기존 vs CTA 추가)와 상세 페이지 진입 여부 사이에는 상관관계가 없다. 즉, 두 그룹의 상세 페이지 진입률(CVR)은 동일하다. $$H_0: p_A = p_B$$  
  
대립가설 ($H_1$): 추천 공모전 카드에 CTA 버튼을 추가하는 것은 상세 페이지 진입 여부에 영향을 미친다. 즉, 실험군(B)의 상세 페이지 진입률이 대조군(A)보다 높을 것이다. $$H_1: p_A < p_B$$

<br>

## Q3-3. 위 가설을 수행하기 위해 어떤 검정 방법을 사용해야하고 이유는 무엇인지 작성해주세요.

## A3-3. 아래 검정 이름과 이유를 채워주세요.
답변: 위 가설에 필요한 검정은 (**카이제곱 검정 (Chi-squared Test)**) 이고,  

이유는 ```본 실험의 종속변수는 '상세 페이지 진입 여부(진입 O / 진입 X)'라는 범주형 변수이고, 독립변수는 '실험 그룹(A: 기존 / B: CTA 추가)'이라는 범주형 변수이다. 두 범주형 변수 간의 연관성(독립성)을 검정해야 하므로, 분할표를 기반으로 하는 카이제곱 독립성 검정이 적합하다. t-test는 두 그룹의 연속형 변수의 평균 차이를 비교할 때 사용하는 것이므로, 전환율(비율)을 비교하는 본 실험에는 카이제곱 검정이 올바른 선택이기 때문``` 이다.

<br>

## Q3-4. 위에서 답한 것을 토대로, 검정을 수행해주세요. (t test와 카이제곱 검정 중)


## A3-4 아래 셀에 코드 작성해주세요. 그리고!!!
그리고 나온 값을 통해 수행하신 a/b test 결과를 설정하신 가설에 맞게 해석해주세요.

In [4]:
import pandas as pd
from scipy.stats import chi2_contingency

# 1. 관측 데이터 (가정)
# 그룹당 약 2,033명씩 배정
# 대조군(A): CVR 12% → 진입 244명, 미진입 1789명
# 실험군(B): CVR 15.8% → 진입 321명, 미진입 1712명
control_results = [244, 1789]   # 대조군(A): [진입 성공, 진입 실패]
variant_results = [321, 1712]   # 실험군(B): [진입 성공, 진입 실패]

# 2. 분할표(Contingency Table) 생성
data = [control_results, variant_results]

# CVR 확인
cvr_a = control_results[0] / sum(control_results)
cvr_b = variant_results[0] / sum(variant_results)
print(f"대조군(A) CVR: {cvr_a:.4f} ({cvr_a*100:.2f}%)")
print(f"실험군(B) CVR: {cvr_b:.4f} ({cvr_b*100:.2f}%)")
print(f"CVR 차이: {(cvr_b - cvr_a)*100:.2f}%p")
print("-" * 50)

# 3. 카이제곱 검정 수행
chi2, p_value, dof, expected = chi2_contingency(data)

# 4. 결과 출력
print(f"카이제곱 통계량: {chi2:.4f}")
print(f"자유도(dof): {dof}")
print(f"P-value: {p_value:.6f}")
print("-" * 50)

alpha = 0.05
if p_value < alpha:
    print(f"결과: P-value({p_value:.6f})가 유의수준 {alpha}보다 작으므로 귀무가설을 기각합니다.")
    print("=> CTA 버튼 추가가 상세 페이지 진입률에 유의미한 변화를 주었습니다.")
else:
    print(f"결과: P-value({p_value:.6f})가 유의수준 {alpha}보다 크므로 귀무가설을 기각할 수 없습니다.")
    print("=> CTA 버튼 추가의 효과가 통계적으로 증명되지 않았습니다.")

대조군(A) CVR: 0.1200 (12.00%)
실험군(B) CVR: 0.1579 (15.79%)
CVR 차이: 3.79%p
--------------------------------------------------
카이제곱 통계량: 11.8728
자유도(dof): 1
P-value: 0.000570
--------------------------------------------------
결과: P-value(0.000570)가 유의수준 0.05보다 작으므로 귀무가설을 기각합니다.
=> CTA 버튼 추가가 상세 페이지 진입률에 유의미한 변화를 주었습니다.


**결과해석**

답변:

카이제곱 통계량은 11.87, p-value는 약 0.000570으로 유의수준 α=0.05보다 훨씬 작으므로 **귀무가설(H₀: 두 그룹의 CVR이 동일하다)을 기각**한다.

- 대조군(A, 기존 UI)의 상세 페이지 진입률(CVR)은 **12.00%**
- 실험군(B, CTA 버튼 추가)의 상세 페이지 진입률(CVR)은 **15.79%**
- CVR 차이: **+3.79%p** (상대적으로 약 31.6% 개선)

따라서 대립가설(H₁)을 채택하며, **추천 공모전 카드에 "자세히 보기" CTA 버튼을 추가하는 것이 상세 페이지 진입률을 통계적으로 유의미하게 향상**시켰다고 결론 내릴 수 있다. 이는 명확한 행동 유도 요소가 유저의 클릭 행동을 촉진한다는 가설을 지지하며, CTA 버튼 도입을 전체 유저에게 배포하는 것이 합리적인 의사결정이다.

<br>

**1주차 기초통계 과제는 여기까지였습니다. 수고하셨습니다!**